### PR TITLE
Eng 1332 fix/docs

### PR DESCRIPTION
**Pull request by sblack4**
https://rhythmic.atlassian.net/browse/ENG-1332

I know the checks are failing but I can't get them to use terraform v0.13 (they're failing bc of 0.12 issues).

### CODE REVIEW

File: lambda/create_jira.py [hunk 1]
```diff
@@ -25,39 +27,48 @@
...
-        'issuetype': {'name': ISSUE_TYPE}
-    }
+        'summary': 'Alert - {}'.format(subject),
+        'description': description,
+        'issuetype': {'name': ISSUE_TYPE}}
 
     issue = jira.create_issue(fields=issue_fields)
     jira.add_comment(
-        issue.key, 'Alert triggered by {} AWS CloudWatch integration'.format(INTEGRATION_NAME))
+        issue.key, 'Alert triggered by {} aws cloudwatch integration'.format(INTEGRATION_NAME))
```
Comment (cdaniluk) [inline lambda/create_jira.py:72]: y?

**Review by cdaniluk — APPROVED** [review]

File: lambda/create_jira.py (hunk 1 above)
Comment (sblack4) [reply lambda/create_jira.py:72]: oops

Comment (sblack4) [reply lambda/create_jira.py:72]: I'm not nearly as good at Vim as I should be and sometimes I mash buttons 🤷

Comment (cdaniluk) [reply lambda/create_jira.py:72]: I always mash buttons

**Pull request merged**